span-stacktrace/README.md

```diff
@@ -40,6 +40,22 @@ SdkTracerProvider tracerProvider = SdkTracerProvider.builder()
 OpenTelemetrySdk sdk = OpenTelemetrySdk.builder().setTracerProvider(tracerProvider).build();
 ```
 
+### Configuration
+
+Even if autoconfiguration is not yet supported, usages of this module must use the
+`otel.span.stacktrace.min.duration` configuration option (defaults to 5ms) to
+allow consistent configuration across usages.
+
+The following constants are provided as a convenience:
+
+- `StackTraceSpanProcessor.CONFIG_MIN_DURATION`
+- `StackTraceSpanProcessor.CONFIG_MIN_DURATION_DEFAULT`
+
+It means default this feature will capture a stacktrace for all spans that last 5ms or more,
+disabling it must be allowed by setting a negative value to `otel.span.stacktrace.min.duration`.
+Setting `otel.span.stacktrace.min.duration` to zero means stacktrace will be captured for all
+spans without span duration limit.
```

Comment:
what do you mean by `without span duration limit`?

Reply:
I mean that every span will have a stack trace without first checking the span duration to know if the stack trace overhead is worth it.

Reply:
so every span. The additional words seem to be irrelevant? If you agree ...
```suggestion
spans.
```